### DISCUSSION TITLE
The weapon barrel is not in a line with the vector of the cursor

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#14256] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

As the title saying.
![Minigun 2](https://github.com/FakeFishGames/Barotrauma/assets/73634685/0e7cfe44-45d5-416f-b2c8-881caf372e10)

There are basically many mods with buttless heavy weapons that have this problem. It also exists in vanilla but it is not obvious.

### Reproduction steps

1. subscribe to https://steamcommunity.com/sharedfiles/filedetails/?id=3279677991
2. type `spawnitem otfx_minigun inventory` in console
3. hold the minigun in hands and observe its barrel and cursor when aiming

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed now in the unstable branch, and the fix will be included in the next update. :)
